policyengine_us/parameters/gov/states/nd/tax/income/credits/marriage_penalty/taxable_income_base.yaml

```diff
@@ -2,13 +2,19 @@ description: North Dakota has a nonrefundable marriage-penalty credit for joint
 values:
   2021-01-01: 12_550
   2022-01-01: 12_950
+  2023-01-01: 13_850
 
 metadata:
   label: marriage-penalty credit taxable income base amount
   unit: USD
+  uprating: gov.irs.uprating
   period: year
   reference:
   - title: 2021 ND income tax form and instruction booklet, Marriage Penalty Credit Worksheet and line 22
-    href: https://revenue.nebraska.gov/files/doc/tax-forms/2021/f_1040n_booklet.pdf#page=16
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2021-iit/2021-individual-income-tax-booklet.pdf#page=16
   - title: 2022 ND income tax form and instruction booklet, Marriage Penalty Credit Worksheet and line 22
-    href: https://revenue.nebraska.gov/files/doc/2022_Ne_Individual_Income_Tax_Booklet_8-307-2022_final_5.pdf#page=16
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2022-iit/2022-individual-income-tax-booklet.pdf#page=16
+  - title: 2023 ND income tax form and instruction booklet, Marriage Penalty Credit Worksheet and line 22
+    href: https://www.tax.nd.gov/sites/www/files/documents/forms/individual/2023-iit/2023-individual-income-tax-booklet.pdf#page=16
+  - title:  North Dakota legal code Section 57-38-01.28
```

Comment:
I dont think this section mentions the threshold - could you double check, also this looks like this is rounded every year, please find the right legal code section

Reply:
still need section